tests/lib/rules/no-triple-slash-reference.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @fileoverview Enforces triple slash references are not used
+ * @author Danny Fritz
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require("../../../lib/rules/no-triple-slash-reference"),
+    RuleTester = require("eslint").RuleTester;
+
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var ruleTester = new RuleTester();
+ruleTester.run("no-triple-slash-reference", rule, {
+
+    valid: [
+        {
```

Comment:
Can you add a valid test that uses comments? I'd just like to be sure the rule doesn't incorrectly warn on double-slash, for example.